### PR TITLE
Dat257 57 Update database with more data

### PR DESCRIPTION
**Pull request by filipkauffeldt**
Add more data for most countries to the database and fix bugs with country repository and csv reader

### CODE REVIEW

File: API/Model/CountryRepository.cs [hunk 1]
```diff
@@ -65,27 +65,29 @@ private int CreateIntHashFromString(string inputString)
             return Math.Abs(ivalue);
         }
 
-        public async Task<Country> AddCountry(Country country)
+        public async Task<Country> AddCountryAsync(Country country)
```
Comment (jakobschy) [inline API/Model/CountryRepository.cs:68]: Looks like the CountryController is still using the AddCountry Function

**Review by jakobschy — CHANGES_REQUESTED** [review]
Branch produces errors when run

**Comment by filipkauffeldt** [comment]
> Branch produces errors when run

Fixed

**Review by jakobschy — DISMISSED** [review]
Looks good

**Review by jakobschy — APPROVED** [review]

**Pull request merged**